src/System.Private.ServiceModel/src/System/ServiceModel/Description/XmlSerializerOperationBehavior.cs

```diff
@@ -364,10 +364,8 @@ internal void EnsureMessageInfos()
                         _request = CreateMessageInfo(this.Operation.Messages[0], ":Request");
                         // We don't do the following check at Net Native runtime because XmlMapping.XsdElementName 
                         // is not available at that time.
-                        bool skipVerifyXsdElementName = false;
-#if FEATURE_NETNATIVE
-                        skipVerifyXsdElementName = GeneratedXmlSerializers.IsInitialized;
-#endif
+                        bool skipVerifyXsdElementName = Fx.IsUap && GeneratedXmlSerializers.IsInitialized;
```

Comment:
If IsUap is true do you always expect GeneratedXmlSerializers.IsInitialized to be true as well?